bareos/files/bareos-config.jinja

```diff
@@ -34,10 +34,11 @@
     {% else %}
         {%- for name, config in resources.iteritems() %}
 {{ type }} {
-            {% if type|lower in requires_name and 'name' not in config|lower %}
+            {%- if type|lower in requires_name and "Name" not in config %}
+
```

Comment:
I think that removing the `|lower` to the `config` dict and forcing an exact comparison like `Name` is a move that might cause other errors: [as the documentation states](http://doc.bareos.org/master/html/bareos-manual-main-reference.html#x1-1180008.3.1), case is resource names is ignored by bareos, so if the pillar is populated with lowercase, or mixed in some other way, these will not match.